### DISCUSSION TITLE
Failed to save email settings, reason: Unknown

### DISCUSSION THREAD

**Discussion by oswaldB** [Budibase/budibase#6488] [Help] [answered]
Hello,

Is there another way to save the email configuration instead of using the UI?

It seems the ui is not working in my case.

Tahnk you

**Comment by melohagan** [comment]
Hi @oswaldB 

Not that I'm aware of - what's your issue?

**Comment by oswaldB** [comment]
Hello,

I got this error when I filled the email screen in the admin section.

I use mailjet and it doesn't seems to work


Any idea ?

Oswald Bernard
________________________________

**Comment by oswaldB** [comment]
Do you have any tips on how i can fix this. 

The app is going for production tomorrow.....

Thank you

**Reply by melohagan** [reply]
Have you tried clicking the "Reset" button and re-entering the email config?

**Reply by aladin-bilalagic** [reply]
Reset is disabled if there is no saved email.
I have the same issue with self-deployment. 

While it works on DigitalOcean droplet, it doesn't work properly out of box when self hosted.

**Comment by oswaldB** [comment]
Yep. I cannot use the reset button.

Can we maybe change it in the code or in the db?

Thank you

Oswald Bernard
________________________________

**Reply by aladin-bilalagic** [reply]
I am trying to solve it for myself, I hope that I'll find solution by the end of the day.

Can you tell me, where is your Budibase instance hosted, what type (docker, kubernetes on premise or cloud?), and which email provider do you attempt to use?

And do you use reverse proxy, if so Caddy of Nginx?

**Comment by oswaldB** [comment, marked as answer]
Hello,

I use mailjet as email providers.

I use docker on a Google cloud instance.

And the reverse proxy is Caddy.

Thanks for the help

Oswald Bernard
________________________________

**Reply by aladin-bilalagic** [reply]
Can you please try setting security type to None/STARTTLS and port to 587 and see if it works? In my case that way it works.
It is an SSL issue it seems.

**Reply by oswaldB** [reply]
It works !!


Thank you so much

**Reply by aladin-bilalagic** [reply]
You are welcome!
